src/main/resources/opensearch_settings.json

```diff
@@ -92,9 +92,43 @@
         },
         "analyzer": "fnr_sok"
       },
+      "alle_aktiviteter_utlopsdatoer": {
```

Comment:
For å unngå å rename parametere kalte jeg disse "alle_" for å vise til at de inkluderer både avtalte og ikke avtalte aktivteter.

Det er også mulig å rename dagens felter til å starte med "avtalte_", men dette er en veldig tung jobb da disse verdiene blir sendt frem og tilbake til frontend